src/Components/Footer/Footer.js

```diff
@@ -8,65 +8,65 @@ const Footer = () => {
         <img src={logo} alt="logo" className="logo-img" data-cy="logo-img"></img>
         <div className="footer-team-container">
           <div className="fe-container">
-            <p data-cy="fe-team">FrontEnd Team:</p>
+            <p data-cy="fe-team" tabIndex={0}>FrontEnd Team:</p>
```

Comment:
Is `tabIndex={0}` the same as `tabIndex="0"`?

Reply:
yeah this is how my vscode suggest its written when I start typing. both work.